integration/integration.go

```diff
@@ -118,15 +118,15 @@ func GetIntegrationName(result string, integration string) string {
 	return res.Data.Name
 }
 
-func GetEcrWithCrossAccountCreds(result string) api.AwsEcrWithCrossAccountIntegration {
+func GetEcrWithCrossAccountCreds(result string) api.AwsEcrIamRoleIntegration {
```

Comment:
curious as to the change from `api.AwsEcrWithCrossAccountIntegration` to `api.AwsEcrIamRoleIntegration`.

Reply:
api.AwsEcrWithCrossAccountIntegration is the v1 implementation

```
type AwsEcrWithCrossAccountIntegration struct {
	commonIntegrationData
	Data AwsEcrDataWithCrossAccountCreds `json:"DATA"`
}

type AwsEcrDataWithCrossAccountCreds struct {
	Credentials AwsCrossAccountCreds `json:"CROSS_ACCOUNT_CREDENTIALS" mapstructure:"CROSS_ACCOUNT_CREDENTIALS"`
	AwsEcrCommonData
}
```

Reply:
I was just curious as to whether the v1 implementation is used anymore. If not could we just convert it to v2? Just think the naming is better in the v1 implementation.
This is a nitpick though...

Reply:
It is no longer used. We will eventually do a clean up in the go-sdk